### PR TITLE
Integration installation: rephrase procedure

### PR DESCRIPTION
**Pull request by c0ffeeca7**

### CODE REVIEW

File: source/docs/use/configuration/basic.md [hunk 1]
```diff
@@ -1,84 +1,61 @@
...
+        You actually need to do this, don't skip it.
 
-!!! tip
-    You actually need to do this, don't skip it.
+3. In the bottom right corner, select **+ Add integration**.
+4. Search for **HACS** and select it.
+5. Acknowledge the statements and select **Submit**.
+6. Authenticate the integration:
+    - HACS uses a device OAuth flow for authentication against the GitHub API.
+    - Copy the device code and select the link [https://github.com/login/device](https://github.com/login/device). **Do not click on submit yet!**
```
Comment (c0ffeeca7) [inline source/docs/use/configuration/basic.md:27]:
```suggestion
    - Copy the device code and select the link [https://github.com/login/device](https://github.com/login/device). **Do not select Submit yet!**
```

File: source/docs/use/configuration/basic.md [hunk 2]
```diff
@@ -1,84 +1,61 @@
...
+- Restarted Home Assistant
 
-1. First go to the "Settings" in the Home Assistant UI
-1. Then "Devices & Services" and the "Integrations" tab in the Home Assistant UI
 
-## Clear your browser cache
+## Installing the HACS integration
 
-Before HACS can show up in the list you need to clear your browser cache or perform hard-refresh.
+1. In Home Assistant, go to <!-- hacs:my info **{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}** -->.
```
Comment (c0ffeeca7) [inline source/docs/use/configuration/basic.md:16]:
```suggestion
1. In Home Assistant, go to <!-- hacs:my integrations **{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}** -->.
```

File: source/docs/use/configuration/basic.md [hunk 3]
```diff
@@ -1,84 +1,61 @@
...
 
-## In your Home Assistant UI go to the Integrations panel
+- [Downloaded HACS](/docs/use/download/prerequisites.md)
+- Restarted Home Assistant
 
-1. First go to the "Settings" in the Home Assistant UI
-1. Then "Devices & Services" and the "Integrations" tab in the Home Assistant UI
 
-## Clear your browser cache
+## Installing the HACS integration
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:14]:
```suggestion
## Setup the HACS integration
```

Alternatively "Configure"

File: source/docs/use/configuration/basic.md [hunk 4]
```diff
@@ -1,84 +1,61 @@
...
+        You actually need to do this, don't skip it.
 
-!!! tip
-    You actually need to do this, don't skip it.
+3. In the bottom right corner, select **+ Add integration**.
+4. Search for **HACS** and select it.
+5. Acknowledge the statements and select **Submit**.
+6. Authenticate the integration:
+    - HACS uses a device OAuth flow for authentication against the GitHub API.
+    - Copy the device code and select the link [https://github.com/login/device](https://github.com/login/device). **Do not select Submit yet.**
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:27]:
```suggestion
    - Copy the device code and select the link [https://github.com/login/device](https://github.com/login/device).
```

There should no longer be a "submit" button to select.

File: source/docs/use/configuration/basic.md [hunk 5]
```diff
@@ -1,84 +1,61 @@
...
-Here you click on "Authorize hacs".
-
----
-
-![image](/assets/images/config_flow/part5.png)
-
-Once you see the confirmation screen, you can close the tab and go back to Home Assistant.
-
----
+    ![image](/assets/images/config_flow/part2.png)
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:51]: No longer there.

File: source/docs/use/configuration/basic.md [hunk 6]
```diff
@@ -1,84 +1,61 @@
...
 
-![image](/assets/images/config_flow/part2.png)
+8. Assign HACS to an area and select **Finish**.
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:53]:
```suggestion
7. Assign HACS to an area and select **Finish**.
```

As current 7 needs to be removed.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: source/docs/use/configuration/basic.md [hunk 7]
```diff
@@ -1,84 +1,57 @@
...
+- Restarted Home Assistant
 
-1. First go to the "Settings" in the Home Assistant UI
-1. Then "Devices & Services" and the "Integrations" tab in the Home Assistant UI
 
-## Clear your browser cache
+## Setting up the HACS integration
 
-Before HACS can show up in the list you need to clear your browser cache or perform hard-refresh.
+1. In Home Assistant, go to <!-- hacs:my info **{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}** -->.
```
Comment (c0ffeeca7) [inline source/docs/use/configuration/basic.md:16]:
```suggestion
1. In Home Assistant, go to <!-- hacs:my integrations **{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}** -->.
```

File: source/docs/use/configuration/basic.md [hunk 8]
```diff
@@ -1,84 +1,57 @@
...
-id: basic
-title: Initial Configuration
-description: 'Basic configuration'
+title: Initial configuration
+description: 'Setting up and authenticating the HACS integration'
 ---
 
-!!! info
-    _This page assumes that you have already completed the [download](/docs/use/download/prerequisites.md)._
+## Prerequisites
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:6]: This should now be the new `!!! prerequisites` admonition

File: source/docs/use/configuration/basic.md [hunk 9]
```diff
@@ -1,84 +1,58 @@
...
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:6]: Is this header still needed?

Comment (c0ffeeca7) [reply source/docs/use/configuration/basic.md:6]: it makes the section show in the sidebar

File: source/docs/use/configuration/basic.md [hunk 10]
```diff
@@ -1,84 +1,58 @@
...
-Before HACS can show up in the list you need to clear your browser cache or perform hard-refresh.
+1. In Home Assistant, go to <!-- hacs:my integrations **{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}** -->.
+2. Clear your browser cache.
+    - Before HACS can show up in the list, you need to clear your browser cache or perform a hard-refresh.
+    !!! tip
+        You actually need to do this, don't skip it.
 
-!!! tip
-    You actually need to do this, don't skip it.
+3. In the bottom right corner, select **+ Add integration**.
```
Comment (ludeeus) [inline source/docs/use/configuration/basic.md:23]:
```suggestion
3. In the bottom right corner, select **+ {{coreui('ui.panel.config.integrations.add_integration')}}**.
```

File: source/docs/use/configuration/basic.md [hunk 11]
```diff
@@ -1,84 +1,58 @@
...
-description: 'Basic configuration'
+title: Initial configuration
+description: 'Setting up and authenticating the HACS integration'
 ---
 
-!!! info
-    _This page assumes that you have already completed the [download](/docs/use/download/prerequisites.md)._
+## Prerequisites
 
+!!! Prerequisites
```
Comment (c0ffeeca7) [inline source/docs/use/configuration/basic.md:8]:
```suggestion
```

File: source/docs/use/configuration/basic.md [hunk 12]
```diff
@@ -1,84 +1,58 @@
...
 
-## In your Home Assistant UI go to the Integrations panel
+    This page assumes that you have completed the following steps:
```
Comment (c0ffeeca7) [inline source/docs/use/configuration/basic.md:10]:
```suggestion
This page assumes that you have completed the following steps:
```

File: source/docs/use/configuration/basic.md [hunk 13]
```diff
@@ -1,84 +1,58 @@
...
 
-1. First go to the "Settings" in the Home Assistant UI
-1. Then "Devices & Services" and the "Integrations" tab in the Home Assistant UI
+    - [Downloaded HACS](/docs/use/download/prerequisites.md)
+    - Restarted Home Assistant
```
Comment (c0ffeeca7) [inline source/docs/use/configuration/basic.md:13]:
```suggestion
   - [Downloaded HACS](/docs/use/download/prerequisites.md)
   - Restarted Home Assistant
```

**Review by ludeeus — APPROVED** [review]

**Pull request merged**